app/src/main/java/com/swent/assos/model/service/impl/AuthServiceImpl.kt

```diff
@@ -4,13 +4,26 @@ import com.google.android.gms.tasks.Task
 import com.google.firebase.auth.AuthResult
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.auth.FirebaseUser
+import com.swent.assos.config.Config
 import com.swent.assos.model.service.AuthService
 import javax.inject.Inject
 import kotlinx.coroutines.channels.awaitClose
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.callbackFlow
 
 class AuthServiceImpl @Inject constructor(private val auth: FirebaseAuth) : AuthService {
+
+  // TODO: if you are using the firestore emulator, make sure to change the host and port to match
```

Comment:
The keyword TODO is maybe not the best in this context, it is more an advice for future implementations. You could replace it by `NB`